### DISCUSSION TITLE
Getting message "The Control is not on the form" and control is disabled for editing some times.

### DISCUSSION THREAD

**Discussion by Sanjay151993** [khoait/DCE.PCF#54] [General]
When we try to update the record in offline mode we are getting "The Control is not on the form" error and disabled for editing.

**Comment by khoait** [comment]
Hi @Sanjay151993, what type of relationship are you using? 

Please note that it is a limitation for N:N relationship as it is not supported in Mobile Offline Mode. Please see here for more details:
[https://learn.microsoft.com/en-us/power-apps/mobile/offline-capabilities#limitations](https://learn.microsoft.com/en-us/power-apps/mobile/offline-capabilities#limitations)

**Reply by Sanjay151993** [reply]
I am using N:N relationship.

**Reply by Sanjay151993** [reply]
I have one more question, in offline if we try to update polylookup it is working fine, but the same is not working for create scenario. may I know the reason.

**Reply by khoait** [reply]
It is the limitation of Dataverse, we can't add related records until the record is saved. so it's the limitation of the control as well. Please see wiki here for workaround: https://github.com/khoait/DCE.PCF/wiki/PolyLookup#selected-items. 

However, the workaround is meant for online mode. it's not applicable for offline mode at the moment.

Hope it helps!
